README.md

```diff
@@ -312,6 +326,8 @@ API | Description | Auth | HTTPS | Link |
 | adresse.data.gouv.fr | Address database of France, geocoding, and reverse | No | Yes | [Go!](https://adresse.data.gouv.fr) |
 | Battuta | A (country/region/city) in-cascade location API | `apiKey` | Yes | [Go!](https://battuta.medunes.net) |
 | Bing Maps | Create/customize digital maps based on Bing Maps data | `apiKey` | Yes | [Go!](https://www.microsoft.com/maps/) |
+| City Context | Crime, school, and transportation data for US cities | `apiKey` | Yes | [Go!](https://www.citycontext.com/api-reference#/) |
+| CitySDK | Open APIs for select European cities | No | Yes | [Go!](http://www.citysdk.eu/citysdk-toolkit/) |
```

Comment:
Can you confirm that this is free? Couldn't quickly find any documentation saying one way or the other.

Reply:
CitySDK is free and has no API key required.

City Context is free for 100 calls, and then the price is 0/month for two of their plans: https://www.citycontext.com/plans

I'm guessing that's a typo. If 100 calls is too small, I can remove it.

Reply:
@davemachado I believe this is the only open question here. Shall I remove City Context?

Reply:
It still has a free tier, so no problem 😄